### PR TITLE
Issue-1857: Upgrade swagger dependencies from 1.5.18 to 1.6.2

### PR DESCRIPTION
**Pull request by roshinisebastian**
This PR addresses [#Issue-1857](https://github.com/strongbox/strongbox/issues/1857)

Incremented swagger core version from 1.5.18 to 1.6.2.

### CODE REVIEW

**Comment by carlspring** [comment]
Hi!

Thank you for your contribution! We have tested it and it works as expected!

Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

Also, please, feel free to join our [chat channel](https://chat.carlspring.org/), if you'd like to learn more about the project and/or like to find out what else you could help with.

Kind regards,

Martin

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1857-SNAPSHOT</version>
```
Comment (carlspring) [inline pom.xml:9]:
```suggestion
    <version>1.0-SNAPSHOT</version>
```
Reverting to base version after [successful tests in Jenkins](https://jenkins.carlspring.org/blue/organizations/jenkins/strongbox%2Fbuilds%2Fstrongbox/detail/PR-1888/2/pipeline).

**Comment by roshinisebastian** [comment]
@carlspring I have emailed the ICLA. Thank you.

**Comment by carlspring** [comment]
Thank you! Congratulations on your first pull request! :)

**Pull request merged**